requirements.txt

```diff
@@ -1,3 +1,3 @@
 dead-instrumenter==0.0.3
-ccbuilder==0.0.5
+ccbuilder==0.0.8
```

Comment:
`0.0.9`?

Reply:
Maybe we even want to create 0.0.10 to include the worktree pruning.

Reply:
I'll update all versions (including diopter) once this PR and the other repos have stabilized.